autoPyTorch/datasets/base_dataset.py

```diff
@@ -40,6 +45,17 @@ def type_check(train_tensors: BASE_DATASET_INPUT, val_tensors: Optional[BASE_DAT
             check_valid_data(val_tensors[i])
 
 
+def get_tensor_size(data_tensor: BASE_DATASET_INPUT) -> int:
```

Comment:
Do you think we can add a small test to make sure this flag is_small_preprocess is asserted only when it makes sense?

For example, this variable is true if a np.array of 10 elements is provided but not true if a np.array of 30000 elements is provided. Just to make sure we don't break this in the future.

Reply:
Will do that tomorrow